### ISSUE TITLE
Implement counterclockwise rotation

### ISSUE THREAD

**Issue by vince267** [hjfreyer/pentris#11]
Self explanatory based off the title, but I want to mention the reasoning too. This game is very similar to NES tetris, which I love, and has a ton of cool spins you can do, such as T-spins, line spins, z/s-spins, etc. Part of what makes pentris so fun for me is the spins you can do with the new pieces, such as b/p-spins or spins with the 2-mino (bimino?). However, the lack of counterclockwise rotation makes it impossible to do spins in certain directions. Implementing cc rotation would be super cool to see how much the added flexibility improves gameplay! 

As a suggestion, I'd tie it to the shift button so that one-handed play is still possible. You could also tie clockwise and counterclockwise to the z/c buttons for people who prefer two-handed play, but I am not one of those people so I'd love it on the shift bar.  For a bonus implementation that I don't personally care about, but would make everyone happy here, you could add the option for users to choose their own button layout.

**Comment by hjfreyer** [comment]
Thanks for all the feedback! This one was quite simple so I figured I'd just bang it out. 

I mapped it to Shift+Up, since Shift is quite commonly used to mean "do the opposite" during keyboard shortcuts. Requiring the player to hold two keys may be a drag though, so give it a shot, and if you don't like it, feel free to reopen the bug!

BTW, lemme blow your mind here... a 2-mino is called... a domino!

**Comment by vince267** [comment]
This is awesome, thanks a ton!! Early thoughts are that it feels a bit awkward to do when I want to rotate at the same time as moving left/right, but I’ll try it for a few days and see if I get used to it. Having the option is great for doing new spins though, so I’m happy regardless! 

You’re right, though, it did kinda blow my mind…but not at first lol. I initially read it as if it were pronounced “dough-mino” and I thought “oh uh okay cool”. But then I realized it’s pronounced like the game, and was like, “oh actually that is pretty cool”. And then I realized that the pieces in the dominos game are themselves 2-minos, and hence dominos, and hence the game, and that’s when my brain exploded.
